bootstrap-salt.sh

```diff
@@ -238,7 +238,11 @@ _ECHO_DEBUG=${BS_ECHO_DEBUG:-$BS_FALSE}
 _CONFIG_ONLY=$BS_FALSE
 _PIP_ALLOWED=${BS_PIP_ALLOWED:-$BS_FALSE}
 _PIP_ALL=${BS_PIP_ALL:-$BS_FALSE}
-_SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/etc/salt}
+if uname -a | grep FreeBSD > /dev/null; then
+    _SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/usr/local/etc/salt}
+else
+    _SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/etc/salt}
+fi
```

Comment:
I think overriding default values for `_SALT_ETC_DIR` and `_PKI_DIR` should be done later, let it be in `__configure_freebsd_pkg_details()` function, for the sake of keeping related code close to where it would be used.
So, no need for an `if-else` statement here, it would look much cleaner.